shared-bindings/bitmaptools/__init__.c

```diff
@@ -868,7 +868,55 @@ STATIC mp_obj_t bitmaptools_dither(size_t n_args, const mp_obj_t *pos_args, mp_m
     return mp_const_none;
 }
 MP_DEFINE_CONST_FUN_OBJ_KW(bitmaptools_dither_obj, 0, bitmaptools_dither);
+// requires all 5 arguments
 
+//| def draw_circle(
+//|     dest_bitmap: displayio.Bitmap, x0: int, y0: int, radius: int, value: int
+//| ) -> None:
+//|     """Draws a circle into a bitmap specified using a center (x0,y0) and radius r.
+//|
+//|     :param bitmap dest_bitmap: Destination bitmap that will be written into
+//|     :param int x0: x-pixel position of the circle's center
+//|     :param int y0: y-pixel position of the circle's center
```

Comment:
I think we should use `x` and `y` for these two argument names instead of `x0`, and `y0` since there is only 1 point passed in instead of multiple. If user needed to pass `x0, x1, y0, y1` for instance then having them numbered would be good.

Reply:
Yes, that is a good point. Thanks

Reply:
please see https://github.com/adafruit/circuitpython/pull/7782/commits/6ad053e40375169e907dc5f4949775bef3c6b332